### ISSUE TITLE
Worst accuracy `0.02` compared to keras

### ISSUE THREAD

**Issue by rakesh4real** [pytorch/pytorch#45735]
Hi, I came across this kernel where Pytorch loss is worse than Keras. Can anyone let me know why?

Link to kernel: [click here](https://www.kaggle.com/daishu/answer-why-is-keras-better-than-pytorch)

**Comment by colesbury** [comment]
The link you provided suggests an answer: the loss functions are different.
